backend/wallet/types.py

```diff
@@ -35,6 +35,7 @@ class TransactionType(str, Enum):
     EXTERNAL = "external"
     INTERNAL = "internal"
     OFFCHAIN = "offchain"
+    REFUND = "refund"
```

Comment:
refund type is odd, because it overlaps with above 3 types (a refund can be external / internal / offchain, technically offchain has overlap with external without explanation, as they are shortcut of external_non_offchain, external_offchain)
so probably should add a new field.

Reply:
Yeah maybe makes more sense to rename "external" to somethign that indicates below $1000 txn